src/main/java/io/wispforest/owo/serialization/Endable.java

```diff
@@ -0,0 +1,11 @@
+package io.wispforest.owo.serialization;
+
+/**
+ * Interface that indicates that the object has an end operation to such
+ */
```

Comment:
IMHO it would be nicer to just make an interface that extends AutoCloseable and just makes `close` not throw anything, kind of like [`InfallibleCloseable` in gadget](https://github.com/wisp-forest/gadget/blob/1.20.2/src/main/java/io/wispforest/gadget/util/InfallibleClosable.java).

Reply:
Possibly a good idea as such exists just as a way to prevent issues with calling close method and `end` is just used due to what glisco used within her version.